examples/hosts/app-integration/external-data/README.md

```diff
@@ -18,59 +17,79 @@ In this case, the Fluid collaboration session serves as a "drafting surface" in
 
 In order to accomplish the goals above, we have split up the responsibilities into a few different pieces:
 
-**External Data Service**
+### External Data Service
 
-Many services that would hold the "source of truth" data offer explicit commit style interfaces (e.g. vi REST call or similar) which are not well suited to rapid updates.
+Many services that would hold the "source of truth" data offer explicit commit style interfaces (e.g. via REST call or similar) which are not well suited to rapid updates.
 However, they often expose third-party integration via REST APIS for querying and manipulating data, as well as webhooks for watching updates to the data.
 
 This repo contains a service that mocks the external "source of truth" data server. This mock service offers a REST API collection and webhook interfaces in `./src/mock-external-data-service`. The API requests served by this "external" service are the following:
 
-1. POST `/register-for-webhook`: Register's the sender's URL to receive notifications when the external task-list data changes. Currently, the Customer Service registers its `/external-data-webhook` endpoint here to be called when data changes.
-2. GET `/fetch-tasks`: Fetches the task list from the external data store. Called by the Fluid client.
-3. POST `/set-tasks`: Updates external data store with new tasks list (complete override). Called by the Fluid client.
-4. POST `/debug-reset-task-list`: Resets the external data to its original contents. Called by the Fluid client.
+1. POST `/register-for-webhook?externalTaskListId=XXX`. Required body parameters: `url` (string). Registers the sender's URL to receive notifications when the external task-list data changes. Currently, the Customer Service registers its `/external-data-webhook` endpoint here to be called when data changes.
```

Comment:
Super nitty nit: I know it was me who suggested putting the task list id in the querystring. I think I should have suggested adding it to the body. Oh well, no biggie 😄 .